_posts/en/newsletters/2020-01-08-newsletter.md

```diff
@@ -82,6 +88,7 @@ projects.
   a transaction in order to be able to spend low-value UTXOs.
 
 {% include linkers/issues.md issues="3647,3814,3351" %}
+[Schnorr/Taproot workshops]: http://localhost:4000/en/schorr-taproot-workshop/
```

Comment:
Agree that there is no place like home, but should be changed before release? :)

Reply:
Fixed.  Thanks @practicalswift for catching that!